### DISCUSSION TITLE
GKD V1.7.0 小米 主题、应用商店 跳过广告失效

### DISCUSSION THREAD

**Discussion by Swanby617** [gkd-kit/gkd#566] [General]
GKD V1.7.0 小米 主题、应用商店 跳过广告失效
![870c51b989373df9585f1c631846b5e](https://github.com/gkd-kit/gkd/assets/164135244/a5371108-5026-41e1-a2de-a1c25e7f62f0)

**Comment by lisonge** [comment]
gkd 现在不提供规则，默认订阅已经停止维护

你的规则谁的就去找谁反馈
